app/views/guidance/show.html.erb

```diff
@@ -64,7 +64,13 @@
     <ul class="govuk-list govuk-list--spaced">
       <li>
         <%= govuk_link_to(
-              signing_off_data,
```

Comment:
We can refactor the related method `signing_off_data` 


@d-a-v-e

Reply:
this has been refactored in #3550